src/StripeTests/Services/Invoices/StripeInvoiceServiceTest.cs

```diff
@@ -151,6 +158,14 @@ public void Pay()
             Assert.Equal("invoice", invoice.Object);
         }
 
+        [Fact]
+        public void PayOptions()
```

Comment:
I don't think we need separate tests for both methods, esp. if one simply calls the other. Can you just update the `Pay` test to add options?